doc/man3/DTLS_set_timer_cb.pod

```diff
@@ -30,7 +38,7 @@ The DTLS_set_timer_cb() function was added in OpenSSL 1.1.1.
 
 =head1 COPYRIGHT
 
-Copyright 2017 The OpenSSL Project Authors. All Rights Reserved.
+Copyright 2000-2025 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
Why 2000-2025 and not 2017-2025? Also copyright years get updated automatically before release so it is not necessary (but of course harmless) to update them in PRs.